### DISCUSSION TITLE
Success': False, 'error': 'Your phone number was not provided in E.164 format, or free SMS are disabled for this country

### DISCUSSION THREAD

**Discussion by raman1438** [HACK3RY2J/Anon-SMS#29] [Q&A]
Hi All,
i m trying to send Anon-SMS using KALI linux with Anon-SMS, i am getting below error.

success': False, 'error': 'Your phone number was not provided in E.164 format, or free SMS are disabled for this country , i search google could not get any clue on this.

please help on this.

**Comment by pablogd731** [comment]
The problem is your country code, which is not supported

**Reply by RamYT3000** [reply]
библиотеку requests забыли добавить в send.py, исправьте пж в некст обнове и разрешите анонимные сообщения в РФ пожалуйста
